### ISSUE TITLE
Small errors in tutorial

### ISSUE THREAD

**Issue by asexto** [graphql-python/graphene#1132]
I'm not completely sure about this, but while I was following the tutorial (https://docs.graphene-python.org/projects/django/en/latest/tutorial-plain/) some parts where not completely clear for me and I had to change two details when importing the schema:

* I didn't seem obvious to me that I should create the ingredients application inside the cookbook project and not at the same level. This got obvious after checking the source code of the tutorial, but adding explicit information about this would make things easier.
* I had problems trying to make Django find the scheme. I had to change some code from the "Hello GraphQL - Schema and Object Types" section:
   * In 'cookbook/ingredients/schema.py' I changed '''from cookbook.ingredients.models import Category, Ingredient"  to '''from .models import Category, Ingredient'''
   * In 'cookbook/schema.py' I changed "import cookbook.ingredients.schema" to "import ingredients.schema"

In fact the code of the demo show differences in these two last places comparing it to the snippets shown in the tutorial. Or maybe I did something wrong, so please correct me if this was my mistake.

Thank you!

**Comment by roelzkie15** [comment]
@asexto 

I do not know but I could clearly see that the tutorial is clear on creating the `ingredients` app inside the cookbook project. 

> We will set up the project, create the following:
> 
>   - A Django project called `cookbook`.
>   - An app within `cookbook` called `ingredients`.

Moreover the code snippet clearly backs it up by these lines:

```
# Set up a new project with a single application
django-admin.py startproject cookbook .  # Note the trailing '.' character
cd cookbook
django-admin.py startapp ingredients
```

Finally on your second problem. Do you get any errors with the current code imports? Usually it is not a best practice to implicitly import a module by using something like `import ingredients.schema` else developer will have a problem tracking where the module came from so it will always be correct to declare it more explicitly. Remember **"Explicit is better that implicit"** in python.

**Comment by asexto** [comment]
Hi Roel! Thank you for your answer.

Right, it was my fault I didn't notice the word "within" in that sentence. Maybe a message more verbose like "Create an app called Ingredients inside the folder of the project cookbook" would be more clear for newbies like myself. But its just a suggestion, my bad I didn't understand what it was meant.

About the second observation, it's a tiny one: the imports of the file cookbook/ingredients/schema.py of the basic tutorial page are not the same than the ones published on GitHub. I had better luck with these last ones, what I saw there solved my problems when defining the schema. Probably my application namespace for some reason is "ingredients" but not "cookbook.ingredients". Sure is my fault, anyway for me the code on GitHub was better suited. (even it is 99,99% the same as the one in the tutorial).

The tutorial has this, but it didn't work for me: from cookbook.ingredients.models import Category, Ingredient
So I did like the source code of the project on GitHub says: from .models import Category, Ingredient
